examples/ExportSystemsToCSV/exportSystemsToCSV.go

```diff
@@ -40,8 +82,34 @@ func main() {
 			system.AmazonInstanceID, system.Os, system.Version, system.AgentVersion, system.Created,
 			system.LastContact}
 
-		for _, tag := range system.Tags {
-			outLine = append(outLine, tag.Name)
+		if isGroups {
+			// for a Groups org, let's retrieve the system groups this system is a member of:
+			// NOTE: there are more associations for a system in a Groups org we may want to list here as well:
+			// Policies, direct Users associations, etc
+			var graphs []jcapiv2.GraphObjectWithPaths
+			for skip := 0; skip == 0 || len(graphs) == searchLimit; skip += searchSkipInterval {
+				graphs, _, err := systemsAPIv2.GraphSystemMemberOf(system.Id, contentType, accept, int32(searchLimit), int32(skip))
+				if err != nil {
+					fmt.Printf("Could not retrieve parent groups for system %s, err='%s'\n", system.Id, err)
```

Comment:
if you do a `continue` here, you can get rid of the else block and un-indent, and this will be a bit more readable.